src/components/LoadingScreen.stories.js

```diff
@@ -0,0 +1,11 @@
+import LoadingScreen from './LoadingScreen'
+  
+export default {
+  title: 'Component/LoadingScreen',
+  component: LoadingScreen,
+}
+
+const Template = args => <LoadingScreen {...args} />;
+
+export const Default = Template.bind({})
+Default.args = {};
```

Comment:
If you do not have any arguments, you can shorten this a bit:

```suggestion
export const LoadingScreenDefault = <LoadingScreen />;
```